freqtrade/persistence.py

```diff
@@ -136,7 +142,9 @@ def check_migrate(engine) -> None:
                     else pair
                     end
                 pair,
-                is_open, {fee_open} fee_open, {fee_close} fee_close,
+                is_open, {fee_open} fee_open, {fee_open_cost} fee_open_cost,
```

Comment:
I think I do not understand the syntax here, why `{fee_open} fee_open` (with fee_open in curly braces), how does it differ from simple `is_open`, which is without curly braces?

Reply:
it's a (multiline) f-string ...
`is_open` has ALWAYS been part of the database.
all others are not - so depending on the "state" of the existing database, the query needs to look different - so it could be 

```sql 
select is_open, fee_open fee_open from trades_bak;
```
or 
``` sql 
select is_open, fee fee_open from trades_bak;

```


You'll need to look at this in the context of the whole method ... then it should make sense

Reply:
> it's a (multiline) f-string ...

ah, it's all in a single f-string... 👍